Gemfile

```diff
@@ -2,7 +2,7 @@ require './prox_gem'
 
 prox_gem 'clock'
 prox_gem 'attribute'
-prox_gem 'null_attr'
+prox_gem 'subst_attr'
```

Comment:
Actually, none of these prox-gems seem relevant. This library doesn't use any of them. See the loader: https://github.com/obsidian-btc/fixture/blob/master/lib/fixture.rb